routes/categories.js

```diff
@@ -1,13 +1,14 @@
 const Router = require('koa-router');
 const Category = require('../models/Category');
 const DBMethods = require('./baseDBMethods');
+const auth = require('../common/auth');
 
 const router = new Router();
 
 router.get('/', DBMethods.list(Category))
   .get('/:id', DBMethods.getById(Category))
-  .post('/', DBMethods.create(Category))
-  .put('/:id', DBMethods.update(Category))
-  .delete('/:id', DBMethods.remove(Category));
+  .post('/', auth.authenticate, auth.validateFRCTeamPermissions, DBMethods.create(Category))
```

Comment:
use router.use instead of copy past that code

Reply:
as u add .post you can add .use with auth middleware and all routes after that will be authenticated

Reply:
Done